### ISSUE TITLE
Allow Hours and Minutes for Shiftchange Times

### ISSUE THREAD

**Issue by jwacalex** [engelsystem/engelsystem#536]
Allow to specify hours and minutes when mass creating shifts

**Comment by MyIgel** [comment]
Where exactly? Do you mean in the "Shift change hours" input?

**Comment by jwacalex** [comment]
@MyIgel yeah. It would be fine to specify either 10,11 for shift change hour or 10:15,11:30,11:45,13:00 if you have a certain pattern/offset

**Comment by MyIgel** [comment]
Ok, i'll have a look :)

**Comment by jwacalex** [comment]
generation should be quite straight forward, first adjusting the regexp to something like ``^((\d{2}(:\d{2})?(,|$))*)`` and then add minutes to generation ;)

(i'm currently testing a pull request, but you answered faster as expected ;))

**Comment by jwacalex** [comment]
@MyIgel i'm unable to test the code, since the database migrations are not in the initial database export. how is the preferred way to migrate the database?
